app/controllers/openid_application_controller.rb

```diff
@@ -94,27 +92,48 @@ def analyze_redis_launch_context
 
   def handle_opaque_token(token_string, aud)
     opaque_token = OpaqueToken.new(token_string, aud)
-    opaque_token.set_payload(fetch_issued(token_string))
-
-    return opaque_token if TokenUtil.validate_token(opaque_token)
-
+    @session = Session.find(hash_token(opaque_token))
+    if @session.nil?
+      opaque_token.set_payload(fetch_issued(token_string))
+      opaque_token.set_ssoi(!opaque_token.static?)
+      return opaque_token if TokenUtil.validate_token(opaque_token)
+    elsif opaque_token.set_ssoi(true)
```

Comment:
- By the time it gets into session items, any opaque token that is not a static token is always ssoi for now.
- Static tokens are authenticated before it gets to the above section at https://github.com/department-of-veterans-affairs/vets-api/blob/744acd59d763abb9f57d8a0f992b0351038c90b0/app/controllers/openid_application_controller.rb#L32

Reply:
While above is true, this was not intentionally setting the value as part of the conditional check.  This has been updated to check specifically for the SSOi IDP value rather than a side-effect default.